src/Windows/Avalonia.Win32/OleDataObject.cs

```diff
@@ -44,9 +44,77 @@ public IEnumerable<string> GetFileNames()
 
         public object Get(string dataFormat)
         {
+            if (dataFormat == ClipboardFormats.GetFormat(2)) // check for CF_BITMAP
```

Comment:
I think that to improve the reading of the code it is better to define a constant.
```csharp
const short CF_BITMAP = 2
```
end using

```csharp
if (dataFormat == ClipboardFormats.GetFormat(CF_BITMAP))
```

Reply:
I added all the standard formats as constants from the [documentation](https://docs.microsoft.com/en-us/windows/win32/dataxchg/standard-clipboard-formats)